Convert tango-astor-feedstock to v1 feedstock

beenje opened the pull request:
This PR converts tango-astor-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.13](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] 🔢 Bumped the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [x] Ensured the license file is being packaged.

beenje committed:
Convert to v1 feedstock and use pixi as conda install tool

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.09.27.05.44.00

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

beenje committed:
Update license path

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

beenje committed:
Update tests

beenje committed:
Update to version 7.6.2

beenje committed:
Do not use pixi as install tool

Missing support for linux-ppc64le:
Cannot solve the request because of: No candidates were found for
      shellcheck *.

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.09.27.05.44.00

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6
